sharding/multiShardCoordinator.go

```diff
@@ -107,6 +107,10 @@ func (msc *multiShardCoordinator) IsInterfaceNil() bool {
 // communicationIdentifierBetweenShards is used to generate the identifier between shardID1 and shardID2
 // identifier is generated such as the first shard from identifier is always smaller or equal than the last
 func communicationIdentifierBetweenShards(shardId1 uint32, shardId2 uint32) string {
+	if shardId1 == core.AllShardId || shardId2 == core.AllShardId {
+		return shardIdToString(core.AllShardId)
```

Comment:
might modify also the shardIdToString in order to return "_ALL" if provided parameter is core.AllShardId

Reply:
fixed